broker-daemon/broker-rpc/wallet-service/commit.js

```diff
@@ -116,24 +96,21 @@ async function commit ({ params, relayer, logger, engines, orderbooks }, { Empty
     logger.debug('Outbound channel already exists', { balance: maxOutboundBalance })
   }
 
-  if (!maxInboundBalance) {
-    const paymentChannelNetworkAddress = await inverseEngine.getPaymentChannelNetworkAddress()
-
-    logger.debug('Creating inbound channel', { balance: convertedBalance })
-
-    try {
-      const authorization = relayer.identity.authorize()
-      await relayer.paymentChannelNetworkService.createChannel({
-        address: paymentChannelNetworkAddress,
-        balance: convertedBalance,
-        symbol: inverseSymbol
-      }, authorization)
-    } catch (e) {
-      // TODO: Close channel that was open if relayer call has failed
-      throw new Error(`Error requesting inbound channel from Relayer: ${e.message}`)
-    }
-  } else {
-    logger.debug('Inbound channel already exists', { balance: maxInboundBalance })
+  const paymentChannelNetworkAddress = await inverseEngine.getPaymentChannelNetworkAddress()
+
+  logger.debug('Creating inbound channel', { balance, symbol, inverseSymbol })
+
+  try {
+    const authorization = relayer.identity.authorize()
+    await relayer.paymentChannelNetworkService.createChannel({
```

Comment:
maybe `createInboundChannel` ? Not sure that it's worth the rename, but it seems like an apt description given the other renames you did.